c2corg_ui/static/partials/contexthelp/associate.html

```diff
@@ -0,0 +1,13 @@
+<p translate>Associate documents (e.g. routes, waypoints, books...etc)</p>
+<ul>
+<li><span translate> Enter a discriminatory part of the document's name. It is possible to search on multiple keywords. For a route, this includes the name of the summit(s) associated to the route.</span></li>
+<li><span translate> A list of documents is shown: click on the document to associate, even if there is only one result.</span></li>
+<li><span translate> For a route, the list of documents contains the routes associated to the selected summit or its sub-summits. Select the desired route.</span></li>
+<li><span translate> It is also possible to enter the document Id (appearing in its URL. Exemple: for the route with https://www.camptocamp.org/routes/57510/fr/triangle-du-pelago-eperon-nw-voie-demenge the Id is 57510).</span></li>
```

Comment:
Exemple => Example
Using the common latin abbreviation "eg." is probably better.